docs/tutorial-script-single.rst

```diff
@@ -0,0 +1,280 @@
+.. _script_single:
```

Comment:
Since this file is not included in the table of contents, it should be marked

```rst
:orphan:
```

on the first line. That'll fix a sphinx warning.